xExtension-ReadingTime/static/readingtime.js

```diff
@@ -23,15 +23,15 @@
          reading_time.words_count = reading_time.flux_words_count(flux_list[i]); // count the words
          reading_time.reading_time = reading_time.calc_read_time(reading_time.words_count, 300); // change this number (in words) to your prefered reading speed
 
+
          if (document.body.clientWidth <= 840) { // in mobile mode, the feed name is not visible (there is only the favicon)
              // add the reading time right before article's title
              // in that case, [Time] - [Title] format is used instead of a "|" (as it looks better and doesn't take much more space)
-             if ( document.querySelector("#" + reading_time.flux.id + " ul.horizontal-list li.item.title a").textContent.substring(0,(reading_time.reading_time + 'm - ').length) != reading_time.reading_time + 'm - ' ) {
-                 document.querySelector("#" + reading_time.flux.id + " ul.horizontal-list li.item.title a").textContent = reading_time.reading_time + 'm - ' + document.querySelector("#" + reading_time.flux.id + " ul.horizontal-list li.item.title a").textContent;
+             if ( $("#" + reading_time.flux.id + " ul.horizontal-list li.item.title a")[0].textContent.substring(0,(reading_time.reading_time + 'm - ').length) != reading_time.reading_time + 'm - ' ) {
```

Comment:
It would be a pity to reintroduce jQuery. What is the problem with `document.querySelector()` ?

Reply:
We have removed jQuery from FreshRSS, except for the statistics page, so if you really need it, you need to explicitly load it

Reply:
Oh, dammit… it seems I did a mistaken git pull from an outdated repo… I didn't see that, that line should not be like that…

Reply:
Fixed :)